synapse/tests/common.py

```diff
@@ -81,6 +84,10 @@ def skipIfNoInternet(self):
         if os.getenv('SYN_TEST_NO_INTERNET'):
             raise unittest.SkipTest('no internet access')
 
+    def skipLongTests(self):
+        if not os.getenv('SYN_RUN_LONG_TESTS'):
```

Comment:
This feels like it should default the other way?  I'd rather a "check it out and run" tries to run everything, and we tweak them "off" when it's not right for the use case.  Probably most intuitive to rename it to "SYN_TEST_SKIP_LONG" or something.

Reply:
This have been renamed to ``SYN_TEST_SKIP_LONG`` to explicitly skip long running tests. The .drone.yml was updated to set this envar to 1 on ``PUSH`` events.